src/Plugin.php

```diff
@@ -87,8 +87,9 @@ public static function wp_head() {
     if (!$gtm_id = get_option('shop_analytics_gtm_id')) {
       return;
     }
+    $embed_url_domain = get_option('shop_analytics_url_domain') ?: 'www.googletagmanager.com';
```

Comment:
See https://github.com/netzstrategen/wordpress-cookie-consent/pull/25#discussion_r923313235
```suggestion
    $embed_url_domain = get_option('shop_analytics_gtm_domain') ?: 'www.googletagmanager.com';
```